3-extra/3-fibonacci.js

```diff
@@ -14,8 +14,19 @@
 */
 
 function generateFibonacciSequence(n) {
-    // TODO
-}
+    if (n < 3) {
+        throw "Error";
```

Comment:
this is good. and might even work in JS, just coz JS is such a weirdo, 
but there is a better/proper way of throwing errors in JS: [Throwing our own errors](https://javascript.info/try-catch#throwing-our-own-errors). 
so, whatever you throw, must be some instance of the Error class.